### PR TITLE
[GD2] Implemented remaining streaming providers

### PR DESCRIPTION
**Pull request by nwalters512**
More work on the march towards feature parity with GD1!

* Adds streaming support for HTML5 (HLS), Livestream, Iframe, Dacast, and RTMP streams
* Migrated HLS and RTMP streams to both use [video.js](http://videojs.com/)
* Removed support for MMS streams
* Added a nice "webcast not supported" message to handle the case where we accidentally set a bad type in the sitevar

Iframes were kind of weird to get working: I either ended up with weird double-escaping if I use the `safe` keyword in the template file, or escaped html entities without it. I ended up going with the latter and just regex-replacing the escaped entities on the client. @fangeugene want to see if you can figure this one out?

### CODE REVIEW

File: react/gameday2/components/embeds/EmbedIframe.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import { webcastPropType } from '../../utils/webcastUtils'
+
+const EmbedIframe = (props) => {
+  const divStyle = {
+    width: '100%',
+    height: '100%',
+  }
+
+  let iframeMarkup = props.webcast.channel
+  iframeMarkup = iframeMarkup.replace(/&lt;/, '<')
```
Comment (nwalters512) [inline react/gameday2/components/embeds/EmbedIframe.js]: ¯\_(ツ)_/¯

**Comment by phil-lopreiato** [comment]
Hey @nwalters512 can you merge in master here? I think we have some livestream events coming up

**Comment by nwalters512** [comment]
Yeah, I can get on that this coming week. It's my school's hackathon this weekend so I'm kinda busy :)

**Comment by fangeugene** [comment]
Build GameDay Lite for your hackathon.

**Pull request merged**